test/Transactions/Orleans.Transactions.Tests/Runners/GoldenPathTransactionTestRunner.cs

```diff
@@ -161,5 +170,119 @@ public virtual async Task MultiWriteToSingleGrainTransaction(string grainStates)
+        [InlineData(30, 4, true, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(30, 5, true, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(30, 2, true, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(30, 3, true, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(30, 4, true, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(30, 5, true, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(30, 2, true, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 3, true, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 4, true, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 5, true, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 2, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 3, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 4, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(30, 5, false, false, ReadWriteDetermination.PerAccess)]
+        // low congestion
+        [InlineData(1000, 2, false, true, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 3, false, true, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 4, false, true, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 2, false, true, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 3, false, true, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 4, false, true, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 2, false, true, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 3, false, true, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 4, false, true, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 2, false, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 3, false, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 4, false, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 5, false, false, ReadWriteDetermination.PerGrain)]
+        [InlineData(1000, 2, false, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 3, false, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 4, false, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 5, false, false, ReadWriteDetermination.PerTransaction)]
+        [InlineData(1000, 2, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 3, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 4, false, false, ReadWriteDetermination.PerAccess)]
+        [InlineData(1000, 5, false, false, ReadWriteDetermination.PerAccess)]
+
+        public virtual async Task RandomizedConsistency(int numGrains, int scale, bool avoidDeadlocks, bool avoidTimeouts, ReadWriteDetermination readwrite)
```

Comment:
Not sure if this should be in the golden path test runner.  If it's expected to succeed consistently, then this is fine.  Just questioning to make sure it's in the right place.

Reply:
Understood, but the test classes are mostly simple cruft code.  We can start with 2, just memory and azure, and add clocked versions later If you'd prefer.